### PR TITLE
[CI] Upgrade to Xcode 10.2

### PR DESCRIPTION
**Pull request by anayini**
Upgrades Travis to Xcode 10.2

## Changes in this pull request

Issue fixed: #159 

### Checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I added an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
- [x] I have reviewed the [contributing guide](https://github.com/plangrid/ReactiveLists/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by jessesquires** [comment]
bruh, you gotta `s/12.1/12.2` for the SDKs

**Comment by anayini** [comment]
bruh

File: .travis.yml [hunk 1]
```diff
@@ -21,11 +21,11 @@ env:
      - WORKSPACE="ReactiveLists.xcworkspace"
      - FRAMEWORK_SCHEME="ReactiveLists"
      - EXAMPLE_SCHEME="Example"
-     - SDK=iphonesimulator12.1
+     - SDK=iphonesimulator12.2
```
Comment (benasher44) [inline .travis.yml]: Just do this, and it'll use the latest by default
```suggestion
     - SDK=iphonesimulator
```

**Comment by anayini** [comment]
The more you know...

**Pull request merged**